platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/ImportMigrationContext.java

```diff
@@ -131,4 +132,8 @@
    * </code>
    */
   public Stream<ImportMigrationEntry> entries(Path path, PathMatcher filter);
+
+  public void setImportedSystemProperties(Properties props);
+
+  public String getImportedSystemProperty(String key);
```

Comment:
❗️ javadoc and possibly annotated as `@Nullable` if it returns `null` when none defined. Also don't forget to document it might throw `SecurityException`.
✏️ I would suggest renaming it `getSystemProperty()` since you are inside the `ImportMigrationContext`